css/properties.json

```diff
@@ -1267,7 +1267,7 @@
     "status": "nonstandard"
   },
   "-webkit-mask-image": {
-    "syntax": "<mask-image> [, <mask-image> ]*",
+    "syntax": "<'mask-image'>",
```

Comment:
`<'mask-image'>` is not the same as `<mask-image>`. Looks like `<mask-image>` is similar, but `<'mask-image'>` can change in future according to the [spec](https://drafts.fxtf.org/css-masking-1/#the-mask-image). It shouldn't affect old syntaxes.